inst/templates/ad_adce.R

```diff
@@ -0,0 +1,157 @@
+# Name: ADCE
+#
+# Label: Clinical Event Analysis Dataset
+#
+# Input: ce, adsl, vs
+library(admiral)
+library(dplyr)
+library(lubridate)
+
+
+# Load source datasets ----
+
+# Use e.g. haven::read_sas to read in .sas7bdat, or other suitable functions
+# as needed and assign to the variables below.
+# For illustration purposes read in admiral test data
+
+data("ce")
+data("admiralvaccine_adsl")
+
+
+adsl <- admiralvaccine_adsl
+ce <- ce
+
+
+# When SAS datasets are imported into R using haven::read_sas(), missing
+# character values from SAS appear as "" characters in R, instead of appearing
+# as NA values. Further details can be obtained via the following link:
+# https://pharmaverse.github.io/admiral/articles/admiral.html#handling-of-missing-values # nolint
+
+ce <- convert_blanks_to_na(ce)
+adsl <- convert_blanks_to_na(adsl)
+
+# Derivations ----
+# Get CE records
+ce01 <- ce %>%
+  filter(CECAT == "REACTOGENICITY")
+
+# Get list of ADSL vars required for derivations
+adsl_vars <- exprs(TRTSDT, TRTEDT)
+
+# Create period dataset - for joining period information onto ce records
+# Need to remove datetime variables as otherwise causes duplicate issues
+
+adslvars <- names(adsl)
+xvars <- adslvars[stringr::str_starts(adslvars, "AP") &
+  stringr::str_ends(adslvars, "DTM")]
```

Comment:
We stay away from brackets.  Can you use `filter()` to subset please

Reply:
oh sry i see what you are doiing here.  can you use `select()` with `starts_with` and `ends_with` from `dplyr`

Reply:
updated as suggested